test/e2e/engine/template.go

```diff
@@ -361,10 +361,10 @@ func Build(cfg *config.Config, masterSubnetID string, agentSubnetIDs []string, i
 		}
 	}
 
-	if config.MSIUserAssignedID != "" {
+	/*if config.MSIUserAssignedID != "" {
```

Comment:
For this repro, we want to make sure we're using system-assigned ID and not user-assigned ID